pubspec.yaml

```diff
@@ -22,6 +22,7 @@ dependencies:
     path: packages/gibsonify_repository
   path_provider: ^2.0.8
   flutter_mailer: ^2.0.0
+  dropdown_search: any
```

Comment:
What is the benefit of using `any` as version? I've seen it often but always wondered about this 😄.

Reply:
`any` will fetch the default version. For our purposes specifying versions should be preferred since it represents the latest version of the dependency that has been verified to work - will change it to reflect this.